example/gluon/word_language_model/README.md

```diff
@@ -28,7 +28,9 @@ python train.py --cuda --tied --nhid 650 --emsize 650 --epochs 40  --dropout 0.5
 ```
 python train.py --cuda --tied --nhid 1500 --emsize 1500 --epochs 60  --dropout 0.65     # Test ppl of 88.42
 ```
-
+```
+python train.py --export-only # hybridize and export model graph (requires mxboard)
```

Comment:
Please add instruction to install mxboard

Reply:
added link and made it outside of the main codes.